rules/numeric-host/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "numeric-host"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bulwark-wasm-sdk = "0.3.0"
+anyhow = "*"
```

Comment:
We probably should stop using `"*"` for this. I'd replace this with `anyhow = "1"`, and at some point we should follow suit everywhere else we've got it too.